src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -53,7 +53,10 @@ const SendRequest = () => {
       myRoles.isOSF
```

Comment:
Need to add isHQ checks for BAC-217 and BAC-218 for the HQ Role to have ability to send forward and backward.